common/api/appui-react.api.md

```diff
@@ -2181,8 +2181,9 @@ export class FloatingViewportContentControl extends ViewportContentControl {
 // @beta (undocumented)
 export interface FloatingViewportContentProps {
     contentId: string;
-    initialViewState: ViewState;
+    initialViewState: ViewStateProp;
     onContextMenu?: (e: React_2.MouseEvent) => boolean;
+    viewportRef?: (v: ScreenViewport) => void;
```

Comment:
Could we use an actual `React.Ref<ScreenViewport>` type here? Even though the `ViewportProps["viewportRef"]` is currently defined as a callback...